.jshintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "node": true,
+
+  "curly": true,
+  "latedef": true,
+  "quotmark": true,
+  "undef": true,
+  "unused": true,
+  "trailing": true
+}
```

Comment:
This has came up on other PRs, but we probably shouldn't have a .jshintrc file in here, since we're using ESLint for our JS linting.